gui.pro

```diff
@@ -216,13 +216,18 @@ unix {
     xdgdesktop.extra = "sed 's|PREFIX|$${PREFIX}|' $$PWD/data/moolticute.desktop > $(INSTALL_ROOT)$$xdgdesktop.path/moolticute.desktop"
     INSTALLS += xdgdesktop
 
+    # install metainfo files
+    metainfo.path = $$PREFIX/share/metainfo
+    metainfo.files = $$PWD/data/moolticute.metainfo.xml
+    INSTALLS += metainfo
+
     # install icons
     iconScalable.path = $$PREFIX/share/icons/hicolor/scalable/apps
-    iconScalable.extra = cp -f $$PWD/img/AppIcon.svg $(INSTALL_ROOT)$$iconScalable.path/moolticute.svg
+    iconScalable.extra = $(INSTALL_FILE) $$PWD/img/AppIcon.svg $(INSTALL_ROOT)$$iconScalable.path/moolticute.svg
     icon32.path = $$PREFIX/share/icons/hicolor/32x32/apps
```

Comment:
sorry for my ignorance here... but what does this resolve to?

Reply:
No worries :slightly_smiling_face: 
```
iconScalable.extra = install -m 644 -p $PWD/img/AppIcon.svg $(INSTALL_ROOT)/$PREFIX/share/icons/hicolor/scalable/apps/moolticute.svg
```
`$PWD` is the working directory and INSTALL_ROOT is a variable that's set on install, for example `~/rpmbuild/moolticute-1.02.0/`. `$PREFIX` is normally `/usr`